tests/saitests/sai_qos_tests.py

```diff
@@ -146,6 +147,22 @@ def get_counter_names(sonic_version):
 
     return ingress_counters, egress_counters
 
+def fill_leakout_plus_one(test_case, src_port_id, dst_port_id, pkt, queue, asic_type):
+    # Attempts to queue 1 packet while compensating for a varying packet leakout.
+    # Returns whether 1 packet was successfully enqueued.
+    if asic_type in ['cisco-8000']:
+        queue_counters_base = sai_thrift_read_queue_occupancy(test_case.client, dst_port_id)
+        max_packets = 100
+        for packet_i in range(max_packets):
+            send_packet(test_case, src_port_id, pkt, 1)
+            time.sleep(1)
+            queue_counters = sai_thrift_read_queue_occupancy(test_case.client, dst_port_id)
+            if queue_counters[queue] > queue_counters_base[queue]:
```

Comment:
looks like just send one pkt one time, can't guarantee the queue counter increase, is that right?